apps/client/src/app/pipes/pipes/mob-name.pipe.ts

```diff
@@ -1,17 +1,23 @@
 import { Pipe, PipeTransform } from '@angular/core';
+
 import { LocalizedLazyDataService } from '../../core/data/localized-lazy-data.service';
-import { I18nNameLazy } from '../../model/common/i18n-name-lazy';
 
 @Pipe({
-  name: 'mobName',
+  name: 'mobName'
 })
 export class MobNamePipe implements PipeTransform {
-  constructor(private data: LocalizedLazyDataService) {}
-
-  transform(id: number): I18nNameLazy {
+  public static getActualMobId(id: number) {
```

Comment:
Why a static method here? I could see this inside the LazyDataService but static is soemthing to avoid in Angular, because Dependency injection is here to handle code that can be reused in different places, while static context cannot be mocked and makes it hard to test and follow the implementations and the dependency graph.

Reply:
lack of understanding about statics in angular